src/pages/DashboardPage/DashboardPage.tsx

```diff
@@ -68,15 +76,39 @@ export const mockEntries: mockData[]= [
 
 const DashboardPage = () => {
     const user = useContext(AuthContext);
+    const navigate = useNavigate();
+    const foundEntries = 
+        mockEntries.filter((entry: mockData) => {
+            return user.id === entry.userId
+        }
+    )
+
+    const handleNavigateToJournalEntryForm = (callback: () => void) => {
+        return () => {
+            callback()
+        }
+    }
 
     return user.isLoggedIn ? (
         <main className="dashboard-main">
             <Typography variant="h2">Welcome back {user.displayName}!</Typography>
-            <Box className='dashboard-journal-entries-container'>
-                {mockEntries.map((entry: mockData) => {
-                   return <JournalEntryCard entry={entry} key={entry.id}/>
-                })}  
-            </Box>
+            {foundEntries.length ? 
+            (
+                <Box className='dashboard-journal-entries-container'>
+                    {foundEntries.map((entry: mockData) => {
+                        return <JournalEntryCard entry={entry} key={entry.id}/>
+                    })}
+                </Box>
+            ) : (
```

Comment:
Not sure about the ternary inside of a ternary, should definitely circle back later to check out other solutions